examples/render/quad_render/main.rs

```diff
@@ -164,15 +133,15 @@ fn main() {
     });
 
     pipeline_desc.attributes.push(pso::AttributeDesc {
-        location: 1,
+        location: 0,
```

Comment:
that is reverting @msiglreith change?

Reply:
Should be actually applying it on the `quad_render` example copy.